x-pack/legacy/plugins/apm/public/components/app/ServiceMap/index.tsx

```diff
@@ -28,35 +25,8 @@ interface ServiceMapProps {
   serviceName?: string;
 }
 
-const cytoscapeDivStyle = {
-  background: `linear-gradient(
-  90deg,
-  ${theme.euiPageBackgroundColor}
-    calc(${theme.euiSizeL} - calc(${theme.euiSizeXS} / 2)),
-  transparent 1%
-)
-center,
-linear-gradient(
-  ${theme.euiPageBackgroundColor}
-    calc(${theme.euiSizeL} - calc(${theme.euiSizeXS} / 2)),
-  transparent 1%
-)
-center,
-${theme.euiColorLightShade}`,
-  backgroundSize: `${theme.euiSizeL} ${theme.euiSizeL}`,
-  margin: `-${theme.gutterTypes.gutterLarge}`,
-  marginTop: 0
-};
-const BetaBadgeContainer = styled.div`
-  right: ${theme.gutterTypes.gutterMedium};
-  position: absolute;
-  top: ${theme.gutterTypes.gutterSmall};
-  z-index: 1; /* The element containing the cytoscape canvas has z-index = 0. */
-`;
-
 export function ServiceMap({ serviceName }: ServiceMapProps) {
   const license = useLicense();
-  const { search } = useLocation();
```

Comment:
While you are at it, would you mind removing the references to uifilters since it seems service maps doesn't use them. The `useFetcher` call below can be simplified to something like:

```ts
  const { data } = useFetcher(() => {
    const { start, end, environment, serviceName } = urlParams;
    if (start && end) {
      return callApmApi({
        pathname: '/api/apm/service-map',
        params: {
          query: {
            start,
            end,
            serviceName,
            environment
          }
        }
      });
    }
  }, [urlParams]);
```

Reply:
Yes but uiFilters will need to be an empty string since it's required by the type.

```
 query: {
    environment?: string | undefined;
    serviceName?: string | undefined;
} & {
    uiFilters: string;
} & {
    start: string;
    end: string;
}
```

Reply:
Is it not possible to remove `uiFilters` from the backend?

https://github.com/elastic/kibana/blob/f47022a41d09e241fa8c49bfd4d84d4fa6913deb/x-pack/plugins/apm/server/routes/service_map.ts#L27

Reply:
Going to do this in a separate PR. Will open issue.

Reply:
Created #60754 for this.